frontends/web/src/routes/buy/components/buyTags.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2022 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useTranslation } from 'react-i18next';
+import style from './buyTags.module.css';
+
+export const BestDeal = () => {
+  const { t } = useTranslation();
+  return (
+    <div className={[style.tag, style.bestDeal].join(' ')}>{t('buy.exchange.bestDeal')}</div>
```

Comment:
nit: since it's always 2 template literal would be better to read imo i.e. `${style.tag} ${style.bestDeal}` also in the other palces.

Reply:
sure.